mimic/model/lstm_model.py

```diff
@@ -0,0 +1,80 @@
+from mimic.model.model import Model
+
+from keras.preprocessing.sequence import pad_sequences
+from keras.layers import Embedding, LSTM, Dense, Dropout
+from keras.preprocessing.text import Tokenizer
+from keras.callbacks import EarlyStopping
+from keras.models import Sequential
+import keras.utils as ku 
+import tensorflow as tf
+
+import numpy as np
+import string, os 
+
+class LSTMModel(Model):
+
+    def __init__(self):
+        self.tokenizer = Tokenizer()
+
+    ''' Create sequences to train the model - each is a sequence of tokens that represent characters '''
+    def learn(self, text):
```

Comment:
What format should `text` be in? It might be worthwhile for @Annnnnieee and @jackattackyang to take a look at this as well so we're all on the same page.

Reply:
Just a plain string